packages/swapper/src/types.ts

```diff
@@ -318,6 +320,37 @@ type TradeQuoteBase = {
   swapperName: SwapperName // The swapper that generated this quote/rate
 }
 
+export type SwapId = string
+
+export type SwapMetadata = {
+  lifiRoute?: Route
+  chainflipSwapId?: number
+  stepIndex: SupportedTradeQuoteStepIndex
+}
+
+export enum SwapStatus {
+  Pending = 'pending',
```

Comment:
We will want a notion of `initialized` of sorts too.

Following our discussion, pending seems to really refer to draft/idle currently, and we're missing one status here?